task_download.py

```diff
@@ -0,0 +1,147 @@
+import argparse
+import requests
+import json
+import os
+import logging
+from concurrent.futures import ThreadPoolExecutor
+from datetime import datetime
+
+# constants
+BATCH_URL = 'https://api.scale.com/v1/batches'
+TASK_URL = 'https://api.scale.com/v1/tasks'
+MAX_WORKERS = 200
```

Comment:
I imagine this would be frowned upon by our platform team, let's make it 25 just for good measure.